new/LIT6713DerYaq.xml

```diff
@@ -25,11 +25,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <sourceDesc>
               <listWit>
 <witness corresp="BerOrFol117">Staatsbibliothek Berlin or. fol. 117, p. 413-426 </witness>
-<witness corresp="">Halle Yb 4°7, f. 59ra-60va</witness>
-<witness corresp="BLorient616">British Library Orient. 616</witness>
-<witness corresp="BLorient618">British Library Orient. 618</witness>
-<witness corresp="EMML3442">EMML 3442</witness>
-<witness corresp="EMML4533">EMML 4533</witness>
+<witness corresp="ULBYb47">Halle Yb 4°7, f. 59ra-60va</witness>
+<witness corresp="BLorient616">British Library Orient. 616, f. 156ra-160ra</witness>
```

Comment:
should this now be added to BLorient616 ms_1.5? and BLorient618 ms_1.6? There it is called "Homily of Jacob of Sarug on the Annunciation of the Blessed Virgin" - can this title be mentioned here too?